Documentation/General/SupportedPlatforms.md

```diff
@@ -43,19 +43,29 @@ Compilers:
 

```

Comment:
Update the Wiki instead:
https://github.com/ezEngine/ezEngine/wiki/Supported-Platforms